llvm.proj

```diff
@@ -68,11 +68,12 @@
     <_LLVMBuildArgs Include="-DLLVM_BUILD_LLVM_C_DYLIB=OFF" />
     <_LLVMBuildArgs Include="-DLLVM_ENABLE_DIA_SDK=OFF" />
     <_LLVMBuildArgs Include="-DLLVM_INCLUDE_TESTS=OFF" />
-    <_LLVMBuildArgs Include='-DLLVM_TARGETS_TO_BUILD="X86%3BAArch64%3BARM"' />
+    <_LLVMBuildArgs Include='-DLLVM_TARGETS_TO_BUILD="X86%3BAArch64%3BARM%3BWebAssembly"' />
```

Comment:
curious, what's the size diff on clang etc. when adding a target? I wonder if it'd be worthwhile to condition this based on the target OS (e.g. iOS doesn't need ARM anymore)

Reply:
For the record, as discussed on Discord, ARM codegen target is between 100KB and 5MB of disk cost per binary; WebAssembly is 100KB-3MB.